mobile_config/src/mobile_radio_tracker.rs

```diff
@@ -192,7 +201,9 @@ pub async fn get_tracked_radios(
             entity_key,
             hash,
             last_changed_at,
-            last_checked_at
+            last_checked_at,
+            asserted_location::bigint,
```

Comment:
Can the migration add this column as a `BIGINT` to begin with?

Reply:
Done